local-infra.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+REPO_DIR="$SCRIPT_DIR"
+
+CMD="${1:-}"
+if [[ "$CMD" == "" ]]; then
+  printf "Must specify command, either 'up', 'down', or 'rm'\n" 1>&2
+  exit 1
+fi
+
+case "$CMD" in
+up)
+  docker-compose -f "$REPO_DIR/docker-compose.yaml" -f "$REPO_DIR/docker-compose.postgres.yaml" up --build -d
+  PGPORT=$(docker port bytecodealliance-registry-db-1 5432 | sed -e 's/.*://g')
```

Comment:
I think it may be better to just set a non-5432 port to avoid conflicts, and just suggest to use an additional docker-compose override file to change the port if necessary. That could remove the complication of getting the random port here. Although keeping the random value generation for passwords/keys seems like more appropriate practice to follow.

Reply:
Created `.env` for defining default port numbers.